packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -173,6 +174,14 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
         super._optionChanged(args);
     }
   }
+
+  updateTextAreaAria(emptyViewId: string | null): void {
```

Comment:
```suggestion
  updateInputAria(aria: Record<string, unknown>): void {
      this._textArea.option('inputAttr', aria);
  }
```

Reply:
Or we can do it more common and name it `updateInputAttr(atts)`. It could work with any attribute, not just ARIA-specific ones.

Reply:
Let's leave the naming associated with Aria.

Reply:
It seems to me that it is ideologically more correct to form the aria object inside the messagebox.

Reply:
[604f5dc44f8c0d730eca6fcecfa778976849c4d0]